pandas/src/datetime/np_datetime_strings.c

```diff
@@ -695,7 +753,7 @@ parse_iso_8601_datetime(char *str, int len,
     if (sublen >= 2 && isdigit(substr[0]) && isdigit(substr[1])) {
         out->sec = 10 * (substr[0] - '0') + (substr[1] - '0');
 
-        if (out->sec < 0 || out->sec >= 60) {
```

Comment:
@kawochen pointed this out - unless I'm misunderstanding something they will always be false.